production/ksonnet/loki/query-scheduler.libsonnet

```diff
@@ -0,0 +1,58 @@
+
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+{
+  local container = k.core.v1.container,
+
+  // Override frontend and querier configuration
+  local override = {
+      frontend: {
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+      frontend_worker: {
+          frontend_address: '',
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+  },
+  _config +: {
+      loki: std.mergePatch($._config.loki, override),
+  },
+
+  query_scheduler_args::
+    $._config.commonArgs {
+      target: 'query-scheduler',
+      'log.level': 'debug',
+    },
+
+  query_scheduler_container::
+    container.new('query-scheduler', $._images.query_frontend) +
+    container.withPorts($.util.defaultPorts) +
+    container.withArgsMixin(k.util.mapToFlags($.query_frontend_args)) +
+    $.jaeger_mixin +
+    // sharded queries may need to do a nonzero amount of aggregation on the frontend.
```

Comment:
The scheduler components shouldn't need any resource changes regardless of sharding (the increases were to allow the frontend to do more aggregation work in the logql engine).